### PR TITLE
Added Russian Demo Page

### PR DESCRIPTION
**Pull request by toha87**
This is an example how we can implement different languages

### CODE REVIEW

**Comment by temichpilot** [comment]
This is pretty interesting. Good job Anatoly. Are you actually changing the language from English to Russian?

File: src/containers/RussianDemoPage/RussianDemoPage.js [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+        command: ["Зеркало зеркало ты говоришь на английском", "Зеркало зеркало на стене говори на английском"],
+        answer: "Yes! I know English",
+        func: () => {
+            SpeechRecognitionHook.changeLanguage('en-AU');
+            SpeechRecognitionHook.setLangVoice(0);
+        }
+    };
+
+    const kakDela = {
+        command: ["зеркала зеркала как твои дела", "Зеркала зекрала на стене как твои дела"],
```
Comment (temichpilot) [inline src/containers/RussianDemoPage/RussianDemoPage.js:19]: typo: Зеркало зеркало  instead of  зеркала зеркала

Comment (toha87) [reply src/containers/RussianDemoPage/RussianDemoPage.js:19]: its interesting but when you say this command it recognizes it as зеркала :) idk why

**Comment by toha87** [comment]
> This is pretty interesting. Good job Anatoly. Are you actually changing the language from English to Russian?

Yes, I do change it to Russian so it recognizes and speaks Russian then on specific command it goes back to English

**Review by NikolasGraham — APPROVED** [review]

**Pull request merged**